packages/.prettierrc

```
@@ -0,0 +1,6 @@
+{
+    "tabWidth": 4,
```

```suggestion
    "tabWidth": 4,
```
is 4 not too much? I'd suggest two but it's subjective.

---

It's for Move code.